opentelemetry-sdk/src/opentelemetry/sdk/_metrics/__init__.py

```diff
@@ -148,15 +148,36 @@ def create_observable_up_down_counter(
 
 
 class MeterProvider(APIMeterProvider):
-    """See `opentelemetry._metrics.MeterProvider`."""
+    r"""See `opentelemetry._metrics.MeterProvider`.
+
+    Args:
```

Comment:
Also add remaining args?

Reply:
I ran into issues where documenting `metric_readers` created a cascade of `py:class reference target not found: opentelemetry.sdk._metrics.metric_reader.MetricReader`. Let me try again though

Reply:
fixed but left some todos to avoid more documentation changes in this PR